Mixpanel/InAppNotifications.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  InAppNotifications.swift
+//  Mixpanel
+//
+//  Created by Yarden Eitan on 8/9/16.
+//  Copyright © 2016 Mixpanel. All rights reserved.
+//
+
+import Foundation
+
+protocol InAppNotificationsDelegate {
+    func markNotification(_ notification: InAppNotification)
```

Comment:
The delegate does not have to mark a notification or track a notification. Rather, get notified that when that happened to do extra work. Do you think it makes sense to rename these to `onNotificationMarked` and `onNotificationTracked`?